### DISCUSSION TITLE
The runtime-deps src for dotnet 5 with alpine 3.14 amd64 currently unavailable

### DISCUSSION THREAD

**Discussion by hxiao-godaddy** [dotnet/dotnet-docker#3166] [Q&A] [answered]
Is this something on the radar? 

Currently, the 5.0.10-alpine3.14-amd64 link in the [Linux amd64 Tags table](https://hub.docker.com/_/microsoft-dotnet-runtime-deps) even points to the src of dotnet 3.1 runtime-deps.

**Comment by mthalman** [comment, marked as answer]
That's correct and it's by design.  The image produced by building that runtime-deps Dockerfile located in the 3.1 directory is shared by all versions (3.1, 5.0, 6.0). In other words, we build the image once and tag it for all those versions.  So there's only one Dockerfile.  There are several benefits to this:
* It improves the efficiency of the build process because it only needs to build one image for all those versions.
* It allows for layer sharing for consumers creating more efficiency downstream. Once you've pulled runtime-deps:5.0-alpine, that cached layer can be reused if you attempt to pull runtime-deps:3.1-alpine.

**Reply by hxiao-godaddy** [reply]
Thanks Matt for the answer!
